WeCantSpell.Hunspell/WordList.Query.cs

```diff
@@ -1896,15 +1896,15 @@ private bool CompoundReplacementCheck(ReadOnlySpan<char> word)
                     if (replacementEntry.Med is { Length: > 0 })
                     {
                         // search every occurence of the pattern in the word
-                        var rIndex = word.IndexOf(replacementEntry.Pattern.AsSpan());
+                        var rIndex = word.IndexOf(replacementEntry.Pattern);
```

Comment:
With the more aggressive inlining, maybe it's better to have a single line calling IndexOf with it just starting at a 0 offset.